lang/ilias_zh.lang

```diff
@@ -7939,7 +7939,7 @@ chatroom_adm#:#client_settings#:#客户端设置
 chatroom_adm#:#https#:#HTTPS
 chatroom_adm#:#chat_enabled#:#启用聊天
 chatroom_adm#:#enable_osd#:#启用屏幕通知
-chatroom_adm#:#osd_intervall#:#刷新间隔
+chatroom_adm#:#osd_interval#:#刷新间隔
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.